lib/LocalRepository.js

```diff
@@ -50,7 +50,7 @@ var LocalRepository = function (options) {
     this.ts = new TranslationSet(this.sourceLocale);
 };
 
-var xliffFileFilter = /-[a-z][a-z][a-z]?(-[A-Z][a-z][a-z][a-z])?(-[A-Z][A-Z])?/;
+var xliffFileFilter = /-[a-z][a-z][a-z]?(-[A-Z][a-z][a-z][a-z])?(-[A-Z][A-Z])?.*\.xliff$/;
```

Comment:
This expression needs to be updated. The 2.0 xliff file of webOS is located under the app's name directory. The file name of xliff is `locale.xliff` (i.e ko-KR.xliff) . We don't combine app name with  `-`
something like:
settings /
        └──ko-KR.xliff
        └──en-US.xliff